ocaml/idl/datamodel_host.ml

```diff
@@ -1926,6 +1937,7 @@ let t =
             ~doc_tags:[Networking] "address"
             "The address by which this host can be contacted from any other \
              host in the pool"
+        ; field ~qualifier:DynamicRO ~ty:Bool ~default_value:(Some (VBool false)) "https_only" "Reflets wheather port 80 is open (false) or not (true)"
```

Comment:
Please add new fields to the end of the list. Also there are spelling mistakes in the doc string.

Reply:
I think you also need to add `~lifecycle:[]` here to get datamodel_lifecycle.ml updated properly.